### DISCUSSION TITLE
How to disable this error/warning?

### DISCUSSION THREAD

**Discussion by limaceous-bushwhacker** [LazyVim/LazyVim#4291] [Q&A] [answered]
```
Mason package path not found for **debugpy**:
- `/venv/bin/python`
You may need to force update the package.
Press ENTER or type command to continue
```

I don't use Mason, have it disabled.

**Comment by rogeriofrsouza** [comment, marked as answer]
That's because LazyVim will try to get the package path through Mason: https://github.com/LazyVim/LazyVim/blob/12818a6cb499456f4903c5d8e68af43753ebc869/lua/lazyvim/plugins/extras/lang/python.lua#L109

Are you using the Python extra? You could try to override the default config:
```lua
return {
      "mfussenegger/nvim-dap-python",
      config = function()
          require("dap-python").setup("path/to/debugpy"))
        end
},
```

**Reply by limaceous-bushwhacker** [reply]
Thank you! That works as expected.
